### PR TITLE
Nicer graphs

### PR DESCRIPTION
**Pull request by tomato42**
### Description

Make the graphs more useful, don't hide outliers, provide zoomed in versions of the ecdf and scatter plots.
### Motivation and Context

usability

### Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/tomato42/tlsfuzzer/blob/master/CONTRIBUTING.md) document and my PR follows [change requirements](https://github.com/tomato42/tlsfuzzer/blob/master/CONTRIBUTING.md#change-requirements) therein
- [x] the changes are also reflected in documentation and code comments
- [x] all new and existing tests pass (see Travis CI results)
- [x] [test script checklist](https://github.com/tomato42/tlsfuzzer/wiki/Test-script-checklist) was followed for new scripts
- [x] new test script added to `tlslite-ng.json` and `tlslite-ng-random-subset.json`
- [x] new and modified scripts were ran against popular TLS implementations:
  - n/a
- [x] required version of tlslite-ng updated in requirements.txt and README.md

---
This change is [Reviewable](https://reviewable.io/reviews/tomato42/tlsfuzzer/722)

### CODE REVIEW

**Comment by tomato42** [comment]
I'll fix the TODO in #717

**Review by Jakuje — APPROVED** [review]
looks good. Just a one nit in the comments.

File: tlsfuzzer/analysis.py [hunk 1]
```diff
@@ -711,23 +741,25 @@ def _write_summary(self, difference, p_vals, sign_p_vals, worst_pair,
...
-            txt_file.write(txt)
-            txt_file.write('\n')
+            self._write_stats("Mean", low_mean, mean, high_mean, txt_file)
+            self._write_stats(
+                "Median", low_median, median, high_median, txt_file)
+            self._write_stats(
+                "Trimmed mean (5%)", low_trim_mean_05, trim_mean_05,
+                high_trim_mean_05, txt_file)
+            self._write_stats(
+                "Trimmed mean (25%)", low_trim_mean_25, trim_mean_25,
```
[the remark this answers is not in the document]
Comment (tomato42) [reply tlsfuzzer/analysis.py:781]: good catch, trimmed is the more common name, so it should be trimmed, fixed

**Review by Jakuje — APPROVED** [review]
looks good.

**Pull request merged**